kotest-framework/kotest-framework-api/src/jvmMain/kotlin/io/kotest/core/coroutines/TestScopeContainer.kt

```diff
@@ -0,0 +1,13 @@
+package io.kotest.core.coroutines
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestScope
+import kotlin.coroutines.CoroutineContext
+
+@ExperimentalCoroutinesApi
+class TestScopeContainer(val testScope: TestScope): CoroutineContext.Element {
```

Comment:
Can this be marked internal? Otherwise I suppose it could be used to retrieve the TestScope, albeit unwieldy..

Reply:
No, because it is being used in `kotest-framework-engine` (other module)

Reply:
I think `@KotestInternal` is another option then :)